Maxmin.js

```diff
@@ -35,4 +35,4 @@ function minmax(a, b, c) {
     largest(a, b, c);
 
 }
-minmax(3, 4, 6);
\ No newline at end of file
+minmax(5, 4, 6);
```

Comment:
i think it's ok to return one of the two numbers if they are equal, you don't need the last else statement.